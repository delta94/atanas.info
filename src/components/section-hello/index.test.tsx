import * as React from 'react';
import { shallow } from 'enzyme';

import { SectionHello } from '.';

describe('SectionHello component', () => {
	it('Should render the SectionHello component', () => {
		const wrapper = shallow(<SectionHello />);

		expect(wrapper).toMatchSnapshot();
	});
});
