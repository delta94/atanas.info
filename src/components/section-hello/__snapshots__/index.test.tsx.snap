// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHello component Should render the SectionHello component 1`] = `
<Component
  hasButton={false}
  hasShell={true}
  id="hello"
>
  <div
    className="c-canvas"
    id="canvas"
  />
  <Component />
</Component>
`;
